v19.1/managed-connect-to-your-cluster.md

```diff
@@ -7,50 +7,65 @@ build_for: [managed]
 
 Once your Managed CockroachDB cluster is available and you've [set your password](managed-sign-up-for-a-cluster.html#set-your-password) and [signed in](managed-sign-up-for-a-cluster.html#sign-in), you can start interacting with your cluster.
 
-## Before you begin
+## Step 1. Authorize your network
 
-1. On the machine from which you want to connect to the cluster, create a `certs` directory. The `ca.crt` file that you download later will go here.
+Managed CockroachDB requires you to authorize the networks that can access the cluster to prevent denial-of-service and brute force password attacks:
 
-    Any client connecting to the cluster will need access to the `ca.crt` file.
+- In a development environment, you need to authorize your application server’s network and your local machine’s network. If you change your location, you need to authorize the new location’s network, else the connection from that network will be rejected.
+- In a production environment, you need to authorize your application server’s network.
 
-2. Make sure you have a SQL user and password to use in your connection string.
+You can use the Console to authorize networks:
 
-    For details on creating additional users, see [User Management](managed-authorization.html#use-the-console).
-
-## Authorize your network
+1. Navigate to your cluster's [Networking](managed-networking-page.html) page.
 
-Before you connect to your cluster, you need to authorize your network (i.e., whitelist the public IP address of the machine). Otherwise, connections from this machine will be rejected.
+    <img src="{{ 'images/v19.1/managed/networking.png' | relative_url }}" alt="Networking page" style="border:1px solid #eee;max-width:100%" />
 
-Once you are [logged in](managed-sign-up-for-a-cluster.html#sign-in), you can use the Console to authorize your network:
+    The **Networking** page displays a list of authorized networks (i.e., an IP address whitelist) that can access the cluster.
```

Comment:
nit: "IP network whitelist" might be better language, since ranges of IPs are used, not plain IP addresses. (We convert plain IP addresses to CIDR notation on the backend.)